WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -569,11 +613,11 @@ - (void)removeGapMarkerForTopic:(ReaderAbstractTopic *)topic ifNewPostsOverlapMa
 {
     ReaderGapMarker *gapMarker = [self gapMarkerForTopic:topic];
     if (gapMarker) {
-        NSDate *newestPostDate = ((ReaderPost *)newPosts.firstObject).sortDate;
-        NSDate *oldestPostDate = ((ReaderPost *)newPosts.lastObject).sortDate;
-        NSDate *gapDate = gapMarker.sortDate;
+        CGFloat highestRank = [((ReaderPost *)newPosts.firstObject).sortRank doubleValue];
+        CGFloat lowestRank = [((ReaderPost *)newPosts.lastObject).sortRank doubleValue];
+        CGFloat gapRank = [gapMarker.sortRank doubleValue];
```

Comment:
Ah, yes. CGFloat would be a double on 64 bit devices but not those pesky 32 bit devices still lingering out there.